src/overlays/actors/ovl_En_Sellnuts/z_en_sellnuts.c

```diff
@@ -22,99 +98,1072 @@ const ActorInit En_Sellnuts_InitVars = {
     (ActorFunc)EnSellnuts_Draw,
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_80ADD964 = {
-    { COLTYPE_NONE, AT_NONE, AC_ON | AC_TYPE_PLAYER, OC1_ON | OC1_TYPE_ALL, OC2_HIT_PLAYER, COLSHAPE_JNTSPH, },
-    { ELEMTYPE_UNK0, { 0x00000000, 0x00, 0x00 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_ON, },
+static ColliderCylinderInitType1 sCylinderInit = {
+    {
+        COLTYPE_NONE,
+        AT_NONE,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_ON | OC1_TYPE_ALL,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK0,
+        { 0x00000000, 0x00, 0x00 },
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_ON,
+        OCELEM_ON,
+    },
     { 27, 32, 0, { 0, 0, 0 } },
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80ADDB00[] = {
+static ActorAnimationEntryS D_80ADD990[] = {
+    { &D_06005488, 1.0f, 0, -1, 0, 0 },  { &D_0600B0B4, 1.0f, 0, -1, 0, 0 },  { &D_06004AA0, 1.0f, 0, -1, 2, 0 },
+    { &D_06004E38, 1.0f, 0, -1, 2, 0 },  { &D_060029E8, 1.0f, 0, -1, 2, 0 },  { &D_06005CA8, 1.0f, 0, -1, 2, 0 },
+    { &D_060038CC, 1.0f, 0, -1, 0, 0 },  { &D_06003CC0, 1.0f, 0, -1, 0, 0 },  { &D_060012F4, 1.0f, 0, -1, 2, 0 },
+    { &D_06004700, 1.0f, 0, -1, 2, 0 },  { &D_06001BC8, 1.0f, 0, -1, 0, 0 },  { &D_06003438, 1.0f, 0, -1, 2, 0 },
+    { &D_06001E2C, 1.0f, 0, -1, 0, 0 },  { &D_06000994, 1.0f, 0, -1, 0, 0 },  { &D_06002268, 1.0f, 0, -1, 2, 0 },
+    { &D_06002F08, 1.0f, 0, -1, 0, 0 },  { &D_0600577C, 1.0f, 0, -1, 0, -4 }, { &D_060029E8, 1.0f, 8, -1, 2, 0 },
+    { &D_060029E8, 1.0f, 4, -1, 2, -4 }, { &D_060029E8, 1.0f, 0, -1, 2, 0 },  { &D_06001BC8, 1.0f, 0, -1, 0, 0 },
+    { &D_060012F4, -1.0f, 0, -1, 2, 0 }, { &D_06002670, 1.0f, 0, -1, 2, 0 },
```

Comment:
This is an amazing number of animations for this actor, I guess as a haggler he gesticulates more than I realized.